bundles/org.openhab.automation.groovyscripting/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.automation.groovyscripting-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-automation-groovyscripting" description="Groovy Scripting" version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<feature dependency="true">openhab-core-automation-module-script-rulesupport</feature>
```

Comment:
This dependency could be removed when https://github.com/openhab/openhab-core/pull/1725 gets merged.